tests/foreman/smoke/test_ui_smoke.py

```diff
@@ -166,28 +166,32 @@ def test_smoke(self):
 
             # Add YUM repository to content-view
             self.content_views.add_remove_repos(cv_name, [yum_repository_name])
-            self.assertIsNotNone(self.content_views.wait_until_element
-                                 (common_locators["alert.success"]))
+            # TODO:- re-enable once the bug 1191422 is fixed.
+            # self.assertIsNotNone(self.content_views.wait_until_element
+            #                     (common_locators["alert.success"]))
```

Comment:
Try this instead:

    if not bz_bug_is_open(1191422):
        self.assertIsNotNone(...)